msdk-id-sirius/src/main/java/ISirius.java

```diff
@@ -0,0 +1,73 @@
+import de.unijena.bioinf.ChemistryBase.chem.Ionization;
+import de.unijena.bioinf.ChemistryBase.ms.Ms2Experiment;
+import de.unijena.bioinf.ChemistryBase.ms.Peak;
+import de.unijena.bioinf.ChemistryBase.ms.Spectrum;
+import de.unijena.bioinf.sirius.IdentificationResult;
+import de.unijena.bioinf.sirius.Sirius;
+import javafx.util.Pair;
+
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+import java.util.stream.Stream;
+
+/**
+ * Created by evger on 14-May-18.
+ */
+
+public class ISirius {
+
+    public static Pair<double[], double[]> readMsFile(String path) throws FileNotFoundException {
```

Comment:
Instead of this, use MspImportAlgorithm from the msdk-io-msp package. MSP is a simple text format for mass spectra.